src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -193,10 +214,20 @@ private void createActiveUser(User user) {
 
   private void activateUser(User user) {
     if (Boolean.TRUE.equals(user.getActive())) {
-      log.info("{} is up to date.", user.getId());
+      log.info("User with id '{}' is already active.", user.getId());
     } else {
       user.setActive(true);
-      log.info("Updating {}.", user.getId());
+      log.info("Updating User with id '{}'.", user.getId());
```

Comment:
log messages for activate/deactivate should differ somehow

Reply:
I will fix that